arch/risc-v/Kconfig

```diff
@@ -214,6 +214,8 @@ config ARCH_CHIP_MPFS
 	select ARCH_HAVE_PWM_MULTICHAN
 	select ARCH_HAVE_S_MODE
 	select ARCH_RV_CPUID_MAP
+	select ARCH_HAVE_PERF_EVENTS
+	select ARCH_PERF_EVENTS
```

Comment:
but ARCH_PERF_EVENTS shouldn't be selected in Kconfig.

Reply:
Why not? I wanted it to be on always for all mpfs targets, and this does that

Reply:
Kconfig with prompt string is designed to be enabled by defconfig, only Kconfig without prompt string must and should be selected in Kconfig.

Reply:
please see: https://docs.zephyrproject.org/latest/build/kconfig/tips.html#select-statements